include/lsst/utils/Backtrace.h

```diff
@@ -22,8 +22,8 @@
  */
 
 
-#ifndef LSST_UTILS_BACKTRACE_H
-#define LSST_UTILS_BACKTRACE_H
+#ifndef LSST_CPPUTILS_BACKTRACE_H
```

Comment:
While these are arbitrary, I would have thought these would go with both the namespace and the filename.

Reply:
They did used to do so in the initial commit and then I missed them in the commit that reverted the rename.

Reply:
although shouldn't they stay like that if you want them to be moved?

Reply:
Going to cpputils with the aliased namespace.